PY-SDK/public/src/pyaver/aver_client.py

```diff
@@ -95,19 +85,21 @@ async def load(
                 wallet = owner
             elif(isinstance(owner, Keypair)):
                 wallet = Wallet(owner)
-            else:
-                raise Exception('Invalid Owner Passed. Must be Wallet or Keypair')
+            elif(owner is None):
+                wallet = Wallet(Keypair()) # create random keypair if none
```

Comment:
I'm worried about storing a wallet here because I think sometimes the UMA class needs to private key of the owner. However, the wallet cannot retrieve this information